src/exec/src/bin/occlum_exec_client.rs

```diff
@@ -269,14 +271,21 @@ fn main() -> Result<(), i32> {
         .map(|(key, val)| format!("{}={}", key, val))
         .collect();
 
-    let mut sock_file = String::from(args[0].as_str());
-    let sock_file = str::replace(
-        sock_file.as_mut_str(),
+    let mut sockfile = String::from(args[0].as_str());
```

Comment:
What is the meaning of `args[0]`? It seems that a pair of client and server needs to have the same value of `args[0]` to generate the same hash.

Reply:
updated

Reply:
Change the solution. Add a parameter "instance_dir" for both client and server.